src/TLuaInterpreter.cpp

```diff
@@ -17870,3 +17871,128 @@ int TLuaInterpreter::getScroll(lua_State* L)
     lua_pushnumber(L, result);
     return 1;
 }
+
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#getConfig
+int TLuaInterpreter::getConfig(lua_State* L)
+{
+    auto& host = getHostFromLua(L);
+    const bool currentHost = (mudlet::self()->mpCurrentActiveHost == &host);
+    lua_newtable(L);
+    
+    if (host.mpMap && host.mpMap->mpMapper) {
```

Comment:
This check seems unneccessary to me here, except for the show3dMapView value case.
The other values could be retrieved from host, even if no mpMap or mpMapper exist?